modules/internal/String.chpl

```diff
@@ -2116,20 +2112,20 @@ module String {
       if result.isEmpty() then return result;
 
       var i = 0;
-      while i < result.buffLen {
-        var cp: int(32);
-        var nbytes: c_int;
-        var multibytes = (result.buff + i): c_string;
-        var maxbytes = (result.buffLen - i): ssize_t;
-        qio_decode_char_buf(cp, nbytes, multibytes, maxbytes);
+      while i < result.len {
+        const (decodeRet, cp, nBytes) = decodeHelp(buff=result.buff,
+                                                   buffLen=result.len,
```

Comment:
did you miss `buffLen` while merging master here? I don't think this would compile today. `len` is used in the `while` statement above, too